Code/Framework/AzFramework/AzFramework/Device/DeviceAttributeRAM.h

```diff
@@ -25,5 +25,6 @@ namespace AzFramework
         AZStd::string m_name = "RAM";
         AZStd::string m_description = "RAM available to OS in GiB, usually less than the physically installed RAM.";
         float m_valueInGiB = 0.f;
+        AZStd::string m_value = "";

```

Comment:
Pessimation here with assigning empty string.
The default string constructor creates a string of size 0.
```suggestion
        AZStd::string m_value;

```

Reply:
Only reason I did this was so anyone browsing the code could see what the "default" easily, I wasn't sure if some people might assume "0" as the default for a number value.